tests/ui/coherence/coherence-impls-copy.rs

```diff
@@ -2,7 +2,7 @@
 
 use std::marker::Copy;
 
-impl Copy for i32 {}
+impl Copy for char {}
```

Comment:
Question: why do we need to change this? AFAICT, this test does not use `minicore`?

Reply:
Reverting this change